tools/make_icons.py

```diff
@@ -11,7 +11,7 @@
 """
 
 import matplotlib
-matplotlib.use('agg')
+matplotlib.use('agg')  # noqa
```

Comment:
Skip flake8/pep8 issues with this line. (This line causes a warning about imports not at the top of the file.)